lib/platform/gitlab/index.ts

```diff
@@ -730,7 +726,7 @@ export async function getIssueList(): Promise<GitlabIssue[]> {
   if (!config.issueList) {
     const query = getQueryString({
       per_page: '100',
-      author_id: `${authorId}`,
+      scope: 'created_by_me',
```

Comment:
@rarkins do we need to ignore author here too if config request it?

Reply:
The config option is `ignorePrAuthor` so I don't think they should expect it to work for issues too